### PR TITLE
Nuget Proxy

### PR DESCRIPTION
**Pull request by SharpeRAD**
Adds an Alias for setting the Nuget proxy

### CODE REVIEW

**Comment by patriksvensson** [comment]
Will take a closer look at it later but it looks good from what I can see :smile:

File: src/Cake.Common.Tests/Cake.Common.Tests.csproj [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\xunit.runner.visualstudio.2.0.1\build\net20\xunit.runner.visualstudio.props" Condition="Exists('..\packages\xunit.runner.visualstudio.2.0.1\build\net20\xunit.runner.visualstudio.props')" />
```
Comment (patriksvensson) [inline src/Cake.Common.Tests/Cake.Common.Tests.csproj]: Should be removed.

File: src/Cake.Common.Tests/packages.config [hunk 2]
```diff
@@ -4,4 +4,5 @@
   <package id="xunit.abstractions" version="2.0.0" targetFramework="net45" />
   <package id="xunit.core" version="2.0.0" targetFramework="net45" />
   <package id="xunit.extensibility.core" version="2.0.0" targetFramework="net45" />
+  <package id="xunit.runner.visualstudio" version="2.0.1" targetFramework="net45" />
```
Comment (patriksvensson) [inline src/Cake.Common.Tests/packages.config]: Should not be added.

**Comment by patriksvensson** [comment]
It looks good to me, but I don't think we should add support for the xUnit Visual Studio runner. At least not in this commit. I also don't think we should add a build.bat in this commit.

Other than that it looks good to me :smile:

**Comment by SharpeRAD** [comment]
Sorry force of habit! I add both by default now as its nice to have a build shortcut in the solution folder and execute individual tests from the explorer. Should be good to merge now. 

Would you be opposed to me adding them in another PR?

**Comment by patriksvensson** [comment]
No problem! I wouldn't be opposed to adding the visual studio runner, but the bat script feels a little bit redundant in my opinion since it would be a bootstrapper for the bootstrapper.

**Comment by patriksvensson** [comment]
Merged :+1:

**Comment by SharpeRAD** [comment]
YAY! Thanks for the blog mention too :smile:

**Comment by patriksvensson** [comment]
:balloon: :smile: :tada:

**Pull request closed without merging**